tests/integration.rs

```diff
@@ -125,12 +125,12 @@ fn user_supplied_errors_succeeding() {
 
     #[derive(Debug)]
     enum Error {
-        CmdError(cradle::Error),
+        CradleError(cradle::Error),
```

Comment:
Maybe just `Cradle`, since it's already an `Error` enum?

Reply:
Yes, done.